echopype/utils/io.py

```diff
@@ -60,6 +60,9 @@ def save_file(ds, path, mode, engine, group=None, compression_settings=None, **k
     if engine == "netcdf4":
         ds.to_netcdf(path=path, mode=mode, group=group, encoding=encoding, **kwargs)
     elif engine == "zarr":
+        # Ensure that encoding and chunks match
+        for var, enc in encoding.items():
+            ds[var] = ds[var].chunk(enc.get("chunks", {}))
```

Comment:
For my curiosity: is this for when `chunks` has not been specified for a given `var`?

Reply:
This is for ensuring that both dask chunks and encoding chunks match otherwise, you'd get an error around here: https://github.com/pydata/xarray/blob/960f15ce35826b85745adf5a7b5d8f7decbfa16d/xarray/backends/zarr.py#L176